src/Compilers/Core/Portable/InternalUtilities/FatalError.cs

```diff
@@ -63,7 +62,6 @@ internal static class FatalError
             {
                 if (s_nonFatalHandler != value)
                 {
-                    Debug.Assert(s_nonFatalHandler == null, "Handler already set");
```

Comment:
💭 Were we not initializing these in tests previously?

Reply:
We were not. And some tests are failing now, I'm gonna undo this once I figure out the cancallation issue.